source/classification1.md

```diff
@@ -1705,10 +1604,10 @@ prediction_plot = (
                 domain=(rare_cancer["Concavity"].min(), rare_cancer["Concavity"].max())
             ),
         ),
-        color=alt.Color("Class", scale=alt.Scale(range=colors), title="Diagnosis"),
+        color=alt.Color("Class", title="Diagnosis"),
     )
 )
-rare_plot + prediction_plot
+#rare_plot + prediction_plot
 glue("fig:05-upsample-2", (rare_plot + prediction_plot))
```

Comment:
Hmm, for some reason I see the wrong chart in the book I build locally here:

![image](https://user-images.githubusercontent.com/4560057/211354761-3c6074d0-02c4-4ec0-9512-92fb50eac85c.png)

Ah, the reason is that the glue variable is overwritten in the last chart. see my edit just before the exercises for fixing this

Reply:
yep, I fixed that just now before looking at your review :)